### ISSUE TITLE
Bug: MusicBrainz lookup URL is hardcoded to always use https

### ISSUE THREAD

**Issue by unkissedfrog** [whipper-team/whipper#437]
Hi, 
when trying to copy a CD, whipper seems not to be able to establish a network connection (perhups because of trying to resolv the musicbrainz address via https on port 80?): 

> INFO:whipper.command.cd:using configured read offset 99
> INFO:whipper.command.cd:checking device /dev/sr3
> CDDB disc id: 7407300a
> MusicBrainz disc id EXBOGqIv3kyBD7MFtN2djbtQteA-
> MusicBrainz lookup URL https://musicbrainz.org:80	; use MusicBrainz server at host[:port]/cdtoc/attach?toc=1+10+138172+150+15907+31804+45701+57296+72532+83266+95846+105313+126155&tracks=10&id=EXBOGqIv3kyBD7MFtN2djbtQteA-
> Disc duration: 00:30:40.293, 10 audio tracks
> WARNING:whipper.common.program:network error: (NetworkError(),)
> WARNING:whipper.common.program:continuing without metadata
> Submit this disc to MusicBrainz at the above URL.
> 
> INFO:whipper.command.cd:FreeDB identifies disc as ['Airbourne / Boneshaker']
> CRITICAL:whipper.command.cd:unable to retrieve disc metadata, --unknown argument not passed

Passing the --unknown argument works fine. 

I used version 0.9

> whipper -v
> whipper 0.9.0

**Comment by JoeLametta** [comment]
Hi, I think I've identified the cause of this issue: title updated.
Method `getMusicBrainzSubmitURL`, lines 400 & 401:
https://github.com/whipper-team/whipper/blob/8f5559ebc82b66a0a4fe3379368380dd2f5bcc43/whipper/image/table.py#L388-L401

**Comment by Freso** [comment]
This would be a good GCI task, esp. since we have a student churning through GCI whipper tasks. :)

**Comment by ABCbum** [comment]
> This would be a good GCI task, esp. since we have a student churning through GCI whipper tasks. :)
@Freso 😆😆

**Comment by JoeLametta** [comment]
Crosspost (from IRC):

> TL;DR: I think the issue the user experienced was an unrelated network error. Still there are bugs:
> 
> 1. Whipper appears to communicate with MusicBrainz using `musicbrainzngs` always over http
> 2. MusicBrainz submit URL always has `https` as protocol (hardcoded, even when inappropriate. Just graphical issue)
> 3. https://github.com/whipper-team/whipper/blob/57d386e82fd36831ad521a1961b6317dd09c75bd/whipper/command/main.py#L23 always defaults to http: `musicbrainzngs.set_hostname` optional argument `use_https`, only available in untagged `musicbrainzngs` codebase, is never passed (if supported it defaults to `False`)

Bug number 2 is actually the one described in this issue.
